src/components/Culr/culr.component.js

```diff
@@ -61,35 +61,37 @@ export async function getUserAttributesFromCulr(user = {}, createCulrAccountAgen
 
   if (responseCode === 'OK200') {
     attributes.accounts = response.result.Account;
-    const {
-      blocked,
-      userPrivilege,
-      municipalityNumber,
-      municipalityAgencyId
-    } = await getUserInfoFromBorchk(response.result, user);
-    attributes.blocked = blocked || false;
-    attributes.userPrivilege = userPrivilege || [];
-    attributes.municipalityNumber = municipalityNumber;
-    attributes.municipalityAgencyId = municipalityAgencyId;
     attributes.culrId = response.result.Guid || null;
-    return attributes;
-  }
-
+    const userInfo = await getUserInfoFromBorchk(response.result, user);
+    addUserInfoToAttributes(attributes, userInfo);
+  } else {
 // Quick fix for Býarbókasavnið. TODO: clean up.
-  const userInfo = await getUserInfoFromBorchk({}, user);
-  if (userInfo && Object.keys(userInfo).length > 0) {
-    attributes.blocked = userInfo.blocked || false;
-    attributes.userPrivilege = userInfo.userPrivilege || [];
-    attributes.municipalityNumber = userInfo.municipalityNumber;
-    attributes.municipalityAgencyId = userInfo.municipalityAgencyId;
+    const userInfo = await getUserInfoFromBorchk({}, user);
+    if (userInfo && Object.keys(userInfo).length > 0) {
+      addUserInfoToAttributes(attributes, userInfo);
+    }
   }
   return attributes;
 }
 
+/**
+ *
+ * @param attributes
+ * @param userInfo
+ */
+function addUserInfoToAttributes(attributes, userInfo) {
+  attributes.errorBorchk = userInfo.errorBorchk || false;
+  attributes.errorCulr = userInfo.errorCulr || false;
+  attributes.blocked = userInfo.blocked || false;
+  attributes.userPrivilege = userInfo.userPrivilege || [];
+  attributes.municipalityNumber = userInfo.municipalityNumber;
```

Comment:
getUserInfoFromBorchk levner en mulighed for at levere "intet" (try/catch) - det skal der vel tages højde for? Her forventer du at municipalityNumber bare eksisterer?